pkg/controller/servicebindingrequest/planner.go

```diff
@@ -85,6 +85,9 @@ func (p *Planner) Plan() (*Plan, error) {
 	if p.sbr.Spec.BackingServiceSelectors != nil {
 		selectors = append(selectors, *p.sbr.Spec.BackingServiceSelectors...)
 	}
+	if p.sbr.Spec.Services != nil {
```

Comment:
A similar condition for p.sbr.Spec.Applications is missing that should be checked in a similar was as for https://github.com/redhat-developer/service-binding-operator/blob/master/pkg/controller/servicebindingrequest/planner.go#L83

Reply:
https://github.com/redhat-developer/service-binding-operator/pull/414/files#r407707455 
On this, could we please have a chat here ^ @Avni-Sharma @DhritiShikhar